docs/cli-dat/uniqueNumbers.py

```diff
@@ -0,0 +1,34 @@
+#Importing libraries
+import numpy as np
+import datetime
+from twilio.rest import Client
+import numpy as np
+from datetime import datetime
+from pytz import timezone
+from io import StringIO
+
+#Cli secrets (need to add to .gitignore)
+client = Client("", "")
+
+#Fetching messages from cli
+def get_total_exchanged():
```

Comment:
@AlexAndrei98 Can you take a look at this and see if you can return it via text?